admin/spec/models/check-window.spec.js

```diff
@@ -80,82 +80,4 @@ describe('check-window schema', function () {
       done()
     })
   })
-
-  describe('Method getCheckWindow', function () {
```

Comment:
would i be right in assuming that you are removing these as all they do is test the mongoose model (which is pointless IMO)?

do we have tests for the service?

Reply:
@GuyHarwood Methods were moved from models to data.services, and those tests were deleted, not sure how you see this...

Reply:
that's what I had hoped.  There is little value in testing the mongoose models